pkg/images/defaults/flavor_test.go

```diff
@@ -84,6 +84,26 @@ func (s *imageFlavorTestSuite) TestGetImageFlavorFromEnv() {
 	}
 }
 
+func (s *imageFlavorTestSuite) TestOpenSourceImageFlavorDevReleaseTags() {
+	f := OpenSourceImageFlavor()
```

Comment:
I think these tests would be more readable if you assert on the example versions set in `testutils.SetExampleVersion(s.T())`. 

```go
if buildinfo.ReleaseBuild {
     s.Equal("3.0.99.0", f.MainImageTag)
     s.Equal("3.0.99.0", f.CollectorImageTag)
     // ...
}
```
WDYT?

Reply:
Sure, both ways work for me.